src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -12,9 +12,10 @@
 <c:set var="FEEDBACK_STUDENTS"><%=FeedbackParticipantType.STUDENTS.name()%></c:set>
 <c:set var="FEEDBACK_INSTRUCTORS"><%=FeedbackParticipantType.INSTRUCTORS.name()%></c:set>
 
-<div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+<div class = "col-sm-12 margin-bottom-15px padding-15px <%= fqForm.isQuestionHasResponses() ? "alert alert-danger" : "background-color-light-green" %>">
     <div class="margin-bottom-7px">
-        <b class="visibility-title">Visibility</b> (Who can see the responses?)
+        <h4><%= fqForm.isQuestionHasResponses() ? "Changing the visibility after collecting responses is not recommended." : "" %></h4>
+        <b class=<%= fqForm.isQuestionHasResponses() ? "" : "visibility-title" %>>Visibility</b> (Who can see the responses?)
```

Comment:
@whipermr5 Here is a screenshot. I thought the blue title didn't look nice on the red background. Will revert if you feel it unnecesary.
![screen shot 2017-06-07 at 6 00 50 pm](https://user-images.githubusercontent.com/18088721/26878519-647b9848-4bab-11e7-8dc6-f1850c074bfd.png)